engine/gamesys/src/gamesys/components/comp_model.cpp

```diff
@@ -1120,6 +1124,14 @@ namespace dmGameSystem
         dmRender::AddToRender(render_context, &ro);
 
         world->m_InstanceBufferDataLocalSpace.SetSize(instance_write_ptr - world->m_InstanceBufferDataLocalSpace.Begin());
+
+        // Update statistics for the render item
+        if (render_item->m_Buffers->m_LastUsedFrame != world->m_CurrentFrameTick)
+        {
+            world->m_StatisticsVertexDataSize += dmGraphics::GetIndexBufferSize(ro.m_IndexBuffer) + dmGraphics::GetVertexBufferSize(ro.m_VertexBuffers[VX_DECL_BASE_BUFFER]);
+            render_item->m_Buffers->m_LastUsedFrame = world->m_CurrentFrameTick;
```

Comment:
We combine vertex + index buffer memory sizes here. An alternative would be to add a property for index buffers, but I'm not sure it's necessary..